docs/Import-reference-genome.md

```diff
@@ -0,0 +1,47 @@
+# Add new reference genome to the database
+This manual is intended to guide you to load new reference genome(s) to the database. 
+
+### What is **Reference Genome** (also know as reference assembly)?
+A digital nucleic acid sequence database assembled by scientists as a representative example of a species' set of genes. Reference genomes can be accessed online using dedicated browsers such as Ensembl or UCSC Genome Browser. 
+
+### Why does reference genomes matter?
+Some of the views in cBioPortal rely on the reference genome version matching the reference genome version used to analyse the original data (before it was loaded into cBioPortal DB).
+
+## How to load reference genome(s) to the database?
+
+##1. prepare a tab delimited txt file with the following fields:
+1. species: the name of organic species e.g human
+2. name: the name of reference genome eg. hg38
+3. build_name: the name of genome assembly e.g GRCh38
+4. nonN_bases: the total number of non N bases
+5. URL: the URL to download the reference genome 
+6. release_date: the date when reference genome released, in a format of yyyy-mm-dd hh:mm:ss
+
+here is a sample reference genome file:
+```
+#species	name	build_name	nonN_bases	URL	release_date
+human	hg19	GRCh37	2897310462	http://hgdownload.cse.ucsc.edu/goldenPath/hg19	2009-02-01 00:00:00
+human	hg38	GRCh38	3049315783	http://hgdownload.cse.ucsc.edu/goldenPath/hg38	2013-12-24 00:00:00
+mouse	mm10	GRCm38	2652783500	http://hgdownload.cse.ucsc.edu/goldenPath/mm10	2011-12-01 00:00:00
+```
+##2. Using import script to load reference genome(s) data to the database:
```

Comment:
Just noticed that in your previous PR, those 3 reference genomes are included in the database schema. Nice!

Reply:
> 1. Do I have to do this before I load any study, or can I also load it afterward? Or is it best to do this after starting a clean database?

you only need to run importReferenceGenome scripts after you create a new database. You can  add a new reference genome anytime, just make sure your reference_genome.txt file not included those existing reference genomes in the DB.

> 2. Is cBioPortal going to use the reference genome from http://hgdownload.cse.ucsc.edu/goldenPath/* ? Will it actually download the reference genome?

genomic coordinates used by the frontend react component are downloaded form those URLs and saved in a JSON file for a runtime lookup.

> 3. What will happen if I load a study for which I do not have a reference genome?

you can not load any study if you do not have a reference genome used by that study imported to the DB first, you will get an error during loading.

> 4. Do I have to synchronize this with the value of the new property in portal.properties?

Yes, you have to update species, genome.name, and genome.build to include new reference genome.

> 5. Will it overwrite or add new reference genomes? After loading new reference genomes, are my previously loaded studies affected?

you will not be added any reference genomes if already exists. After loading new reference genome, your previously loaded studies will not affected, since cBioPortal only supports one reference genome per study. For example, if you have one study profiled previously using hg19, you would like to reanalyze it using hg38, then you will have to create a new study.